xapian-core/tests/testdata/apitest_cluster.txt

```diff
@@ -0,0 +1,7 @@
+A computer cluster consists of a set of loosely or tightly connected computers that work together so that, in many respects, they can be viewed as a single system. Unlike grid computers, computer clusters have each node set to perform the same task, controlled and scheduled by software.
```

Comment:
These were just a few paragraphs taken from wikipedia and they seemed to serve the purpose well enough. Will it be necessary to change it now? Or can I do that when we test more complex algorithms? Because currently, the tests seem to work as expected with this.